frontend/public/components/events.jsx

```diff
@@ -269,7 +269,7 @@ class EventStream extends SafetyFirst {
 
     return {
       active: !nextProps.fake,
-      loading: !nextProps.fake,
+      loading: nextProps.fake ? false : loading,
```

Comment:
Tested the change and it fixes the bug :+1: 
/lgtm

Reply:
This is simpler as

```js
loading: loading && !nextProps.fake,
```